src/lib/Form/Factory/FormFactory.php

```diff
@@ -543,6 +545,21 @@ public function emptyTrash(
         return $this->formFactory->createNamed($name, TrashEmptyType::class, $data);
     }
 
+    /**
+     * @param TrashSearchData|null $data
+     * @param string|null $name
+     *
+     * @return \Symfony\Component\Form\FormInterface
+     */
+    public function searchTrash(
```

Comment:
we decided to move away from further polluting this FormFactory, either use symfony factory, or create specific one if you need to reuse that form somewhere.

Reply:
I do that this way because of `\EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory` has been already injected. I've changed to use the Symfony FormFactory by set `formFactory` property of above class to be public, instead of make a new dependency in controller. I only have a doubt if the property name should be changed.

Reply:
I would say creating new dependency is a way to go, instead of breaking encapsulation here, even more if we want to move away from this `FormFactory`.

Anyone?